src/platform/nodejs/persist/fileSystemPersist.ts

```diff
@@ -0,0 +1,128 @@
+import * as fs from 'fs';
```

Comment:
Is there still a point of having `ArrayPersist` since this is replacing it?

Reply:
Probably not. It isn't completely no-op, so I'd imagine some users might find it useful to manually hook up to if they don't want us mucking with their filesystem.

Reply:
If users do not want to be storing in the file system, this should be handled in the config.

Reply:
I would rather delete it than add a config. WIll just delete